module_utils/network_lsr/argument_validator.py

```diff
@@ -246,6 +247,62 @@ def _validate_string_min_length(self, value):
             return True
 
 
+class ArgValidatorRouteTable(ArgValidator):
+    def __init__(
+        self,
+        name,
+        required=False,
+        default_value=None,
+    ):
+        ArgValidator.__init__(self, name, required, default_value)
+
+    def _validate_impl(self, value, name):
+        v = None
```

Comment:
What about using a meaningful name for `v`? What does `v` mean? maybe value? But I can see that you are parsing `value` into `v`.. what is the difference between `value` and `v`? I would suggest something like: `parsed_value`. What do you think?

Reply:
I know this is an extended issue on network-role.. not specific to this validator. But I would like to do not introduce more single char variable names.

Reply:
Compared to "value", the "v" is normalized value.

Reply:
I changed `v` into `norm_value`, thanks.